### DISCUSSION TITLE
using 10px fontsize in html global causes all components to have smaller sizes

### DISCUSSION THREAD

**Discussion by cowboydiver** [chakra-ui/chakra-ui#7191] [Q&A]
Hello there, I'm using a base font-size of 10px instead of the default 16px to make rem calculations easier for my custom components. However, when using Chakra components they are now smaller, since they are using rem values with a base of 10px instead of 16px. It seams like I have to override and multiply every single rem value in default theme by 1.6 in order to have the same size components as would otherwise be expected. Is there a smarter way to update rem values in the default theme or do I have to do it manually?

**Comment by moazzam-ali** [comment]
@cowboydiver did you find any good solutions?

**Reply by VikaramjitSinghGorraya** [reply]
@cowboydiver @moazzam-ali  Any solutions?

**Comment by veloware** [comment]
if I understand you correctly, this may help, it's what I have in my theme to achieve 1rem = 10px, but keep the base font size to 16px - 

```
styles: {
    global: {
      html: {
        fontSize: '62.5%',
      },
      body: {
        fontSize: '1.6rem',
      },
    },
  },
```

Ive never had an issue with standard components scaling down with the above

@VikaramjitSinghGorraya  FYI ^^

**Comment by VikaramjitSinghGorraya** [comment]
Hi @veloware , I tried the above code but still facing the issue with component size. Like, the input field becomes smaller.
![Screenshot (119)](https://github.com/user-attachments/assets/5b14eb3a-355a-4bbd-ab44-0d3eea142820)
